airflow/gcp/operators/kubernetes_engine.py

```diff
@@ -144,13 +146,13 @@ class GKEClusterCreateOperator(BaseOperator):
 
     @apply_defaults
     def __init__(self,
-                 project_id,
-                 location,
-                 body=None,
-                 gcp_conn_id='google_cloud_default',
+                 project_id: str,
+                 location: str,
+                 body: Optional[Union[Dict, Cluster]] = None,
```

Comment:
```suggestion
                 body: Union[Dict, Cluster],
```
Take a look at validation in `_check_input()`. I think it's also good idea to create new issue for deal with this validation.

Reply:
Type and default value fixed. I will create issue for `_check_input().`